<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CupParameterGadget</class>
 <widget class="QWidget" name="CupParameterGadget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>168</width>
    <height>276</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>CupParameterGadget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#0a0a0a;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">.QFrame {
  background-color: transparent;
  border: 2px solid white; 
  border-radius: 5px;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_CupInclination">
        <property name="styleSheet">
         <string notr="true">.QLabel {
  color:white;
  background-color:transparent;
} </string>
        </property>
        <property name="text">
         <string>Cup
Inclination</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxCupPlanInclination">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox{
color:white; 
border-radius: 5px;
background-color: transparent;
font:25px 'Arial';
font-weight: bold;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>ยบ</string>
        </property>
        <property name="minimum">
         <number>-99999999</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_CupVersion">
        <property name="styleSheet">
         <string notr="true">.QLabel {
  color:white;
  background-color:transparent;
} </string>
        </property>
        <property name="text">
         <string>Cup
Version</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBox_CupPlanVersion">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox{
color:white; 
border-radius: 5px;
background-color: transparent;
font:25px 'Arial';
font-weight: bold;
}</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="suffix">
         <string>ยบ</string>
        </property>
        <property name="minimum">
         <number>-99999999</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_Superior">
        <property name="styleSheet">
         <string notr="true">.QLabel {
  color:white;
  background-color:transparent;
} </string>
        </property>
        <property name="text">
         <string>Superior
(mm)</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxSuperior">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox{
color:white; 
border-radius: 5px;
background-color: transparent;
font:25px 'Arial';
font-weight: bold;
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>-99999999</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_Medial">
        <property name="styleSheet">
         <string notr="true">.QLabel {
  color:white;
  background-color:transparent;
} </string>
        </property>
        <property name="text">
         <string>Lateral
(mm)</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxMedial">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox{
color:white; 
border-radius: 5px;
background-color: transparent;
font:25px 'Arial';
font-weight: bold;
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>-99999999</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_Posterior">
        <property name="styleSheet">
         <string notr="true">.QLabel {
  color:white;
  background-color:transparent;
} </string>
        </property>
        <property name="text">
         <string>Anterior
(mm)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QSpinBox" name="spinBoxPosterior">
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>55</width>
          <height>40</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <italic>false</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QSpinBox{
color:white; 
border-radius: 5px;
background-color: transparent;
font:25px 'Arial';
font-weight: bold;
}</string>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
        <property name="minimum">
         <number>-99999999</number>
        </property>
        <property name="maximum">
         <number>99999999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
